projects/pt1/python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -1183,23 +1183,26 @@ def ConvTbcModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(9, 4, 5), tu.rand(3, 5, 6), tu.rand(6))
 
 
+# For DQ-Q fake quantization ops
+import torch.ao.quantization.fx._decomposed
+
+
 class Conv2dQInt8ModuleBase(torch.nn.Module):
     def __init__(self, groups=1):
         self.groups = groups
         super().__init__()
 
-    def _forward(self, inputVec, weight, bias):
-        inputVec = torch._make_per_tensor_quantized_tensor(inputVec, 0.01, 7)
-        inputVec = torch.dequantize(inputVec)
-
-        weight = torch._make_per_tensor_quantized_tensor(weight, 0.01, 3)
-        weight = torch.dequantize(weight)
-
-        bias = torch.quantize_per_tensor(bias, 0.0001, 0, torch.qint32)
-        bias = torch.dequantize(bias)
+    def _forward(self, input, weight, bias):
+        input = torch.ops.quantized_decomposed.dequantize_per_tensor.default(
+            input, 0.01, 7, -128, 127, torch.int8
+        )
+        weight = torch.ops.quantized_decomposed.dequantize_per_tensor.default(
+            weight, 0.01, 3, -128, 127, torch.int8
+        )
+        bias /= 0.01 * 0.01  # compensate for iact scale * weight scale
```

Comment:
Since the bias is coming in non-quantized to begin with, I'm not sure I understand why it needs to be normalized by the accumulator scale.

Reply:
I found the bias issue: The problem is the size of the relative rounding error when choosing a float bias between 0 and 1 via `torch.rand()`. The issue is fixed when doing the `/= 0.01 * 0.01` because that increases the bias into the range of 1e4, such that the potential rounding error falls below the necessary threshold.

For this PR, I'm going with your suggestion of just using a quantized bias which sidesteps the problem. I might add some additional tests with an f32 bias in another PR, but that would require taking a closer look at the rounding mode and/or comparison tolerance.

Reply:
Sorry for the late reply. 

The bias rounding error makes sense.

The fake quantized models I've seen are usually stand in replacements for non-quantized versions, so the boundary types are non-quantized (inputs and outputs). Regardless, I'm fine with the tests being a true sample unit of `DQ -> op -> Q`.

Reply:
Ah I see what you mean. I implicitly assumed that our tests represent a single layer from a partitioned graph, because all our tests currently only contain a single operator. There, we would have integer boundary types between the layer partitions, but when assembling the whole model, there would be an additional DQ-Q pair around the composed layer partitions, such that the boundary types of the entire thing would be non-quantized again, as you said.

Reply:
I think single layer is fine for these e2e tests.